### PR TITLE
Improve README, and add missing EOF newlines

### PR DESCRIPTION
**Pull request by CodeBizarre**
This fixes many spelling issues in the README, along with adding syntax highlighting to the code blocks, and slightly changing wording to be more clear in some areas. 

It also adds missing EOF newlines, I was going to dive into the comment strings but since I see work being done for Asyncio now bringing lots of changes and I plan on doing some deeper dives into the actual source, I figured I would hold off on extra changes until I see where that is going/do some contributions on that part myself.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,20 +11,20 @@ Full documentation for this example can be found at [https://msfs2020.cc](https:
...
-# Note the default _time is 2000 as to refreshed at 2s
+# Note the default _time is 2000 to be refreshed every 2 seconds
 aq = AircraftRequests(sm, _time=2000)
-# Use _time=ms where ms is the millsec to refresh data to cash.
-# setting ms to 0 will disable data cashing and allwas pull new data form sim.
-# There is still a timeout of 4 trys with a 10ms delay between checks. 
+# Use _time=ms where ms is the time in milliseconds to cache the data.
+# Setting ms to 0 will disable data caching and always pull new data from the sim.
+# There is still a timeout of 4 tries with a 10ms delay between checks.
 # If no data is received in 40ms the value will be set to -999999
```
Comment (danricho) [inline README.md]: I believe that  -999999 is now None in the latest version.

File: README.md [hunk 2]
```diff
@@ -11,20 +11,20 @@ Full documentation for this example can be found at [https://msfs2020.cc](https:
...
-# Each Requests can be fine tuned by seting the time pram.
```
Comment (danricho) [inline README.md:25]: should "pram" be "param"?

File: README.md [hunk 3]
```diff
@@ -108,20 +108,20 @@ Arguments to pass:
 
 Description: Triggers an event in the simulator
 
-## Runing SimConnect on other system.
+## Running SimConnect on another system.
```
Comment (danricho) [inline README.md]: perhaps these would be better?
... on a separate system.
or
... on a different system.

**Review by danricho — COMMENTED** [review]
Take or leave the suggestions :)

**Comment by CodeBizarre** [comment]
Updated, thanks @danricho

**Comment by odwdinc** [comment]
@CodeBizarre and @danricho Thank you!

**Pull request merged**